### PR TITLE
Add some test

### PR DESCRIPTION
**Pull request by eleidan**
### RSpec
The following tests for Rails part are added:
  - for all models, except for the `User`
  - for `TeamsController`.

All the models with tests are provided with validations.

### Jasmine
Only Team related entities are tested.

### TODO
Add RSpec feature tests for the Team context.
Add User model tests.

@alazarchuk , @epmlys , @JosephBuchma , @rubycop

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (alazarchuk) [inline spec/models/user_spec.rb]: Any tests here?

File: spec/models/vacation_request_spec.rb [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    vacation_request = FactoryGirl.build(:vacation_request, status: 'used')
+
+    expect(vacation_request).not_to be_valid
+  end
+
+  describe '.requested_accepted_inprogress' do
+    let(:user) { FactoryGirl.create :user, :with_vacations_of_all_statuses }
+
+    it 'provides accordingly filtered list of vacation requests' do
+      expect(user.vacation_requests.count).to be(6)
```
Comment (alazarchuk) [inline spec/models/vacation_request_spec.rb]: We usually use .to eq(..) for strings and numbers (http://stackoverflow.com/questions/7156955/whats-the-difference-between-equal-eql-and)

**Comment by alazarchuk** [comment]
In general LGTM. Minor issues should be fixed.

File: spec/models/vacation_request_spec.rb (hunk 2 above)
Comment (eleidan) [inline spec/models/vacation_request_spec.rb]: Done.

File: spec/models/user_spec.rb (hunk 1 above)
Comment (eleidan) [inline spec/models/user_spec.rb]: To be delivered when user handling in BB is fixed.

**Pull request merged**